dashboards/nvidia-gpu/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+sample_dashboards:
+  -
+    category: NVIDIA GPUs
+    id: nvidia-gpu
+    display_name: NVIDIA GPU Dashboard
+    description: |-
+      This dashboard has charts for viewing NVIDIA GPUs when monitored by the GPU enabled Google's Ops Agent. It displays GPU utilization, GPU used memory, active GPU processes, Streaming Multiprocessor (SM) block utilization, SM occupancy, SM pipe utilization, PCIe traffic, and NVLink traffic.
+    related_integrations:
+      - id: nvml
+        platform: GCE
+      - id: dcgm
+        platform: GCE
```

Comment:
These fields are meant to link a dashboard to a specific integration, e.g. https://cloud.google.com/monitoring/agent/ops-agent/third-party

Since we don't have a separate integration for GPU VMs, I think this section should be removed for now.

Reply:
I removed this section. Though I'm not sure I understood the comment. I think we may eventually be getting a doc page there for DCGM.

Reply:
FWIW: The current plan is to have a doc here: https://cloud.google.com/monitoring/agent/ops-agent/third-party-nvidia (that's my working URL, anyway)

Reply:
@EvanSimpson Can you say what it requires for us to get into Monitoring -> Integrations page?

Reply:
The integrations will be added as part of the the Ops Agent release that these will go out with, so we'll just want to make sure all of the copy in the metadata files for nvml/dcgm under [this folder](https://github.com/GoogleCloudPlatform/ops-agent/tree/master/integration_test/third_party_apps_data/applications) are correct.